packages/web-frontend/src/containers/EnlargedDialog/EnlargedDialogContent.js

```diff
@@ -104,23 +104,21 @@ export class EnlargedDialogContent extends PureComponent {
       return null;
     }
 
-    const { name, periodGranularity } = viewContent;
+    const { name, periodGranularity, reference } = viewContent;
 
     if (viewContent.entityHeader === '') titleText = `${viewContent.name}`;
     else if (viewContent.entityHeader)
       titleText = `${viewContent.name}, ${viewContent.entityHeader}`;
     else titleText = `${name}${organisationUnitName ? `, ${organisationUnitName} ` : ''}`;
 
-    const style = {
-      textAlign: 'center',
-      color: isExporting ? DARK_BLUE : WHITE,
-    };
-
     return (
-      <DialogTitle style={style}>
-        <span style={styles.titleText}>{titleText}</span>
-        {periodGranularity && this.renderPeriodSelector()}
-      </DialogTitle>
+      <DialogTitleWrapper
+        titleText={titleText}
+        periodGranularity={periodGranularity}
+        isExporting={isExporting}
```

Comment:
I think `isExporting` is only for deciding the color in `DialogTitleWrapper` right? So maybe decide the color here using `isExporting` and pass the color to `DialogTitleWraper` is better?

Reply:
Good point, will make this change.